sdks/python/scripts/run_tox.sh

```diff
@@ -24,9 +24,10 @@
 
 ###########################################################################
 # Usage check.
-if [[ $# != 1 ]]; then
-  printf "Usage: \n$> ./scripts/run_tox.sh <tox_environment>"
+if [[ $# < 1 || $# > 2 ]]; then
+  printf "Usage: \n$> ./scripts/run_tox.sh <tox_environment> <sdk_location>"
   printf "\n\ttox_environment: [required] Tox environment to run the test in.\n"
+  printf "\n\tsdk_location: [optional] SDK artifact location to install dependencies.\n"
```

Comment:
Not sure I understand the meaning of this flag from the description, and `tox --help` does not document  what `--install` is, should this by chance be passed to `--installpkg`?

Reply:
`--install` behaves the same as `--installpkg` but we should use the documented one. Thanks for pointed out.